ballerina/tests/complex-query-test.bal

```diff
@@ -582,6 +747,41 @@ function testGetArrayTypes() returns error? {
     test:assertEquals(value, expectedData, "Expected data did not match.");
 }
 
+@test:Config {
+    groups: ["query", "query-complex-params"]
+}
+function testGetArrayTypesRecord() returns error? {
```

Comment:
Lets add test cases to return primitive datatypes also. Failure testcases. Scenarios where an error is returned

Reply:
Fixed in 91e3f85